libdnf/rpm/package.cpp

```diff
@@ -268,6 +268,9 @@ std::string Package::get_location() const {
 std::string Package::get_package_path() const {
     Solvable * solvable = get_pool(base).id2solvable(id.id);
     if (auto repo = static_cast<repo::Repo *>(solvable->repo->appdata)) {
+        if (repo->get_type() == repo::Repo::Type::COMMANDLINE) {
```

Comment:
It would be nice to have a description why commandline packages requires a different handling. What about to add a note?